nova/app.py

```diff
@@ -24,30 +26,86 @@
+        (156,968,224,983),  # Crop 7 coordinates
+        (302,1104,357,1119) # Crop 8 coordinates
+    ]
+
+    extracted_data = {}
+
+    keys = [
+        "eye:",
+        "pupil_center_x:",
+        "pupil_center_y:",
+        "pachy_apex_x:",
+        "pachy_apex_y:",
+        "pachy_apex_thickness:",
+        "thinnest_local_thickness:",
+        "pupil_diameter:"
+    ]
 
-    # Do something with the image (content_string)
+    for idx, crop_coordinates in enumerate(crop_coordinates_list):
+        cropped_image = pil_image.crop(crop_coordinates)
+        
+        # Perform OCR on the cropped image region
+        extracted_text = pytesseract.image_to_string(cropped_image, config='--psm 6').strip()
+        
+        # Handle missing - sing
+        if idx == 1 or idx == 2:
+            if '+' not in extracted_text:
+                extracted_text = '-' + extracted_text
+
+        # Handle text recognition common errors
+        extracted_text = extracted_text.replace("p", "")
+        extracted_text = extracted_text.replace("u", "")
+        extracted_text = extracted_text.replace("m", "")
+        extracted_text = extracted_text.replace("o", "0")
+        extracted_text = extracted_text.replace("O", "0")
+        extracted_text = extracted_text.replace("+0", "+0.")
+        extracted_text = extracted_text.replace("+0..", "+0.")
+        extracted_text = extracted_text.replace("-0", "-0.")
+        extracted_text = extracted_text.replace("-0..", "-0.")
+        extracted_text = extracted_text.replace("+1", "+1.")
+        extracted_text = extracted_text.replace("+1..", "+1.")
+        extracted_text = extracted_text.replace("-1", "-1.")
+        extracted_text = extracted_text.replace("-1..", "-1.")
+        extracted_text = extracted_text.replace("+2", "+2.")
+        extracted_text = extracted_text.replace("+2..", "+2.")
+        extracted_text = extracted_text.replace("-2", "-2.")
+        extracted_text = extracted_text.replace("-2..", "-2.")
+
+        # Hande the occuring of additional . error
+        if len(extracted_text) > 5:
+            extracted_text = extracted_text[:-1]
+
+        # Use the key to label the values and make them the right type
+        if idx == 0:
+            extracted_data[idx + 1] = f"{keys[idx]} {extracted_text}"
+        elif idx == 5 or idx == 6:
+            extracted_value = int(extracted_text)
+            extracted_data[idx + 1] = f"{keys[idx]} {extracted_value}"
+        else:
+            extracted_value2 = float(extracted_text)
+            extracted_data[idx + 1] = f"{keys[idx]} {extracted_value2:.2f}"
 
     new_image = f"{content_type},{content_string}"
-    extracted_data = {
-        "eye": "Left",
-        "pupil_center_x": 0.03,
-        "pupil_center_y": 0.07,
-        "pachy_apex_x": 0.00,
-        "pachy_apex_y": 0.00,
-        "pachy_apex_thickness": 367,
-        "thinnest_local_thickness": 346,
-        "pupil_diameter": 3.76
-    }
```

Comment:
This is the expected output format. Please edit the code so the results are stored and returned in this format

Reply:
Also, I just realized we need to extract the "K Max (Front)" values - check the test images. I missed those earlier. 
You can call them "k_max_x" and "k_max_y"